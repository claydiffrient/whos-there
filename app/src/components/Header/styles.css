@import '../shared/common.css';

:root {
  --main: rgba(208, 244, 134, 1);
}

.Header {
  margin: 0;
  background-color: var(--main);
}

.NavItem {
  display: inline-block;
  margin-left: 20px;
}
