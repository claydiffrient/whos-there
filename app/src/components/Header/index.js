import React from 'react';
import { Link } from 'react-router-dom';
import styles from './styles.css';


const Header = () => (
  <div className="row">
    <div className="col-xs-12">
      <ul className={`${styles.Header} row`}>
        <li className={`${styles.NavItem} col-xs-2`}>
          <Link to="/">Home</Link>
        </li>
        <li className={`${styles.NavItem} col-xs-2`}>
          <Link to="/reports">Reports</Link>
        </li>
      </ul>
    </div>
  </div>

);

export default Header;
