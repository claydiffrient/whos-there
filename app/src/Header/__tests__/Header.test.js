/* eslint-env jest */
import { shallow } from 'enzyme';
import React from 'react';
import Header from '../index';

describe('PeopleList', () => {
  it('renders properly', () => {
    const wrapper = shallow(
      <Header />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
