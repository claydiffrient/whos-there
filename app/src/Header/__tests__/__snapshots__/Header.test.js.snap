// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PeopleList renders properly 1`] = `
<div
  className="row"
>
  <div
    className="col-xs-12"
  >
    <ul
      className="Header row"
    >
      <li
        className="NavItem col-xs-2"
      >
        <Link
          replace={false}
          to="/"
        >
          Home
        </Link>
      </li>
      <li
        className="NavItem col-xs-2"
      >
        <Link
          replace={false}
          to="/reports"
        >
          Reports
        </Link>
      </li>
    </ul>
  </div>
</div>
`;
